File: DotNet - RouteSolver/RouteSolver/Service/RemoteServices/GDistanceMatrixElement.cs
/*
 @author Indrit Zeqiri
 @Github https://github.com/ilstarno
 @Linkedin https://www.linkedin.com/in/indrit-zeqiris-3b6b8ba6/
*/
namespace RouteSolver.Service.RemoteServices
{
    
    public sealed class GDistanceMatrixElement
    {
        public GDistanceMatrixElement(GDistanceMatrixAtom duration, string status)
        {
            Duration = duration;
            Status = status;
        }

        public GDistanceMatrixAtom Distance { get; set; }


        public GDistanceMatrixAtom Duration { get; set; }


        public string Status { get; set; }
    }
}

File: Java - RouteSolver/src/service/remoteservices/GDistanceMatrixElement.java
package service.remoteservices;

/**
 * @author Indrit Zeqiri
 * @Github https://github.com/ilstarno
 * @Linkedin https://www.linkedin.com/in/indrit-zeqiris-3b6b8ba6/
 */
public class GDistanceMatrixElement {
	private GDistanceMatrixAtom distance;
	private GDistanceMatrixAtom duration;
	private String status;

	public GDistanceMatrixAtom getDistance() {
		return distance;
	}

	public void setDistance(GDistanceMatrixAtom distance) {
		this.distance = distance;
	}

	public GDistanceMatrixAtom getDuration() {
		return duration;
	}

	public void setDuration(GDistanceMatrixAtom duration) {
		this.duration = duration;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

}